src/api/test-controller/index.js

```diff
@@ -218,49 +218,37 @@ export default class TestController {
         return this._enqueueCommand(DispatchEventCommand, { selector, eventName, options, relatedTarget: options.relatedTarget });
     }
 
-    _getUrls (args) {
-        const [, urls] = args;
+    _prepareCookieArguments (args, isSetCommand = false) {
+        const urlsArg = castArray(args[1]);
```

Comment:
I guess we should add argument validation for common cases:
```js
await t.setCookie();
await t.getCookie(null);
//etc.
```

Reply:
Validation implemented the same way as for another commands of TestController